index.html

```diff
@@ -39,8 +39,161 @@ <h1>I’m Abhishek <br>
        <!-- <li><a href="" class="link"><i class="material-icons">feedback</i></a> </li> -->
         </ul>
          </div>
+</section>
 
 
-</section>   
+<section class="works">
+    <div class="project1">
+        <img src="assets/images/Tonic.png" class="img1" alt="">
+        <div class="task"><h2>Tonic</h2>
+            <div class="list">
```

Comment:
Kindly check and add the background color of `#E5E5E5` between the two sections. you can see the difference in the images show below
Figma design

![Screenshot from 2022-09-23 18-56-39](https://user-images.githubusercontent.com/65233860/192038311-2ac4a752-9f2e-458a-b8b0-2672bcc589ee.png)




Your design

![Screenshot from 2022-09-23 18-56-53](https://user-images.githubusercontent.com/65233860/192038344-8747e513-ba26-4434-a3cc-09c6dcc21416.png)

Reply:
how to add that i tried ??